### DISCUSSION TITLE
Is it possible to filter blogs by multiple tags

### DISCUSSION THREAD

**Discussion by liuxiran** [facebook/docusaurus#5547] [Q&A]
I found that I can filter blogs by a single tag,  Is it possible to filter blogs by multiple tags? 
e.g. I select two tags, then get the blogs contains both of the two tags

**Comment by lex111** [comment]
No, this is not possible, as static site generator we can show posts related with only one specific tag.

**Comment by Josh-Cena** [comment]
It's hard and not supported OOTB, but it's doable in userland. You need to enhance the blog plugin, an example given here for docs: https://github.com/facebook/docusaurus/issues/4492#issuecomment-805041391

In `contentLoaded` you receive `blogTags` as part of `blogContents`, which is a collection of tags containing `name`, `items`, etc. You can create your `TagFilter` page, pass the entire `blogTags` as props, and use `addRoute` to register this to some route. And then it's up to you to render it. (I suggest using URL params to dynamically handle filtering)

To give you some more idea, here's a working example of filtering logic: https://github.com/facebook/docusaurus/blob/main/website/src/pages/showcase/index.tsx

**Comment by slorber** [comment]
It should be possible to build this with client-side filtering, on a page that already includes the full list of blog posts, such as the archive page: https://docusaurus.io/blog/archive

Not sure what kind of UX design you have in mind, and if it can fit the current design of our archive page which is currently ordered by years.
